tensorflow/python/ops/array_ops.py

```diff
@@ -49,6 +49,7 @@
 @@reverse
 @@reverse_v2
 @@transpose
+@@framesig
```

Comment:
Alright, but I am unsure about the directory hierarchy and the need for additional files.